src/actions/tokens.js

```diff
@@ -0,0 +1,42 @@
+import { createActions } from 'redux-actions'
+import { wallet } from '../utils/wallet'
+import { getLikelyContracts, getMetadata, getBalanceOf } from '../utils/tokens'
+
+const WHITELISTED_CONTRACTS = (process.env.TOKEN_CONTRACTS || 'berryclub.ek.near,farm.berryclub.ek.near,wrap.near').split(',');
+
+export const handleGetTokens = () => async (dispatch, getState) => {
+    const { accountId } = getState().account
+
+    await dispatch(tokens.likelyContracts.get(accountId))
+
+    let contracts = [...new Set([...getState().tokens.likelyContracts, ...WHITELISTED_CONTRACTS])].reduce((x, contract) => ({
+        ...x,
+        [contract]: { contract }
+    }), {})
+
+    dispatch(tokens.tokensDetails.set(contracts))
+
+    const account = await wallet.getAccount(accountId)
```

Comment:
we'd better should just create `Account` object directly to avoid extra RPC calls, as there is no special handling needed for 2FA / lockup

Reply:
this also should allow to create it in `getMetadata` and `getBalanceOf` actions directly vs passing it in (which feels a bit weird, I don't think action params are typically used to pass dependencies)

Reply:
done